### PR TITLE
[Doc] [Do-not-Merge] Add a parameter to forcefully close fenced topics

### PR DESCRIPTION
**Pull request by momo-jun**
## Modifications
1. To align with #8561, add a parameter for both broker and standalone pulsar to forcefully close fenced topics.
2. Remove a duplicate parameter from the list of standalone pulsar.

> Note: This doc change will be applied to more historical versions (since 2.7.0) after the doc PR gets approval.

## Preview
image
 
## Documentation
  
- [x] `doc`

### CODE REVIEW

**Comment by momo-jun** [comment]
Hi @massakam @codelipenghui @Anonymitaet, can you please review this PR? Thanks.

Please kindly note that this doc change will be applied to more historical versions (since 2.7.0) after the PR gets approval.

**Review by massakam — APPROVED** [review]
LGTM

File: site2/docs/reference-configuration.md [hunk 1]
```diff
@@ -257,6 +257,7 @@ brokerServiceCompactionThresholdInBytes|If the estimated backlog size is greater
 |isAllowAutoUpdateSchemaEnabled|Allow schema to be auto updated at broker level.|true|
 |schemaCompatibilityStrategy| The schema compatibility strategy at broker level, see [here](schema-evolution-compatibility.md#schema-compatibility-check-strategy) for available values.|FULL|
 |systemTopicSchemaCompatibilityStrategy| The schema compatibility strategy is used for system topics, see [here](schema-evolution-compatibility.md#schema-compatibility-check-strategy) for available values.|ALWAYS_COMPATIBLE|
+| topicFencingTimeoutSeconds | If a topic remains fenced for a certain time period (in seconds), it will be closed forcefully. If set to 0 or a negative number, the fenced topic will not be closed. | 0 |
```
Comment (Anonymitaet) [inline site2/docs/reference-configuration.md:260]:
```suggestion
| topicFencingTimeoutSeconds | If a topic remains fenced for a certain time period (in seconds), it is closed forcefully. If set to 0 or a negative number, the fenced topic is not closed. | 0 |
```

Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true.
https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp

**Review by codelipenghui — DISMISSED** [review]

**Review by Anonymitaet — DISMISSED** [review]

**Comment by momo-jun** [comment]
Thanks for your review and approval. I've applied the same changes to more historical versions since 2.7.0. PTAL.

> Hi @massakam @codelipenghui @Anonymitaet, can you please review this PR? Thanks.
> 
> Please kindly note that this doc change will be applied to more historical versions (since 2.7.0) after the PR gets approval.

**Review by Anonymitaet — APPROVED** [review]

**Comment by Anonymitaet** [comment]
Ping @urfreespace we've merged this code PR, please do not forget to migrate changes, thanks

**Pull request merged**